src/app/config/router/index.js

```diff
@@ -0,0 +1,47 @@
+import React, { lazy } from 'react';
+
+import { nanoid } from 'nanoid';
+import { createBrowserHistory } from 'history';
+import { Navigate, Route, Routes } from 'react-router';
+
+const DeclarationForm = lazy(() => import('../../pages/DeclarationForm'));
+const DeclarationFormWrapper = lazy(() => import('../../Layouts/declarationFormWrapper'));
+
+export const history = createBrowserHistory();
+
+export const routes = {
+	declarationForm: {
+		id: nanoid(4),
+		path: '/declaration-form',
+		exact: true,
+		component: DeclarationForm,
+		layout: DeclarationFormWrapper,
+		content: 'Declaration Form'
+	},
+	default: {
+		id: nanoid(4),
+		path: '/',
+		component: () => <Navigate to={`/declaration-form`} />
+	}
+};
```

Comment:
no idea why we don't use array instead

Reply:
i don't know why code on repo show like that, i have config with eslint and prettier and also githook to but sometime it's still happening (just on repo, my code on local not showing like that -.-)

Reply:
Just created like that to map with object-key-name,
Should i change it to an array?